ADAL/tests/ADAcquireTokenTests.m

```diff
@@ -586,6 +653,111 @@ - (void)testSilentExpiredATBadMRRT
      }];
     
     TEST_WAIT;
+    
+    // verify telemetry output
+    // there should be 2 telemetry events recorded as there are 2 acquire token calls
+    XCTAssertEqual([receivedEvents count], 2);
+    
+    // the following properties are expected for the 1st acquire token call
+    XCTAssertEqual([self adGetPropertyCount:[receivedEvents firstObject]
```

Comment:
Pull out `[receivedEvents firstObject]` into a local variable and use that instead.